jmix-flowui/flowui-themes/src/main/resources/META-INF/resources/themes/jmix-lumo/component/jmix-horizontal-menu.css

```diff
@@ -42,17 +62,35 @@
   height: var(--lumo-icon-size-s);
 }
 
-vaadin-menu-bar-list-box .jmix-horizontal-menu-content-item .text-component:not(:last-child) {
-  margin-right: var(--lumo-space-l);
-}
-
 .jmix-horizontal-menu-content-item .suffix-component {
   margin-inline-start: auto;
 }
 
-/*to remove padding for '>' sign when a suffix exists to retain all the suffixes in a column*/
-.jmix-horizontal-menu-item-wrapper:not([dir='rtl'])[aria-haspopup='true'][suffixed]::after {
-  padding-left: 0;
+vaadin-menu-bar .jmix-horizontal-menu-content-item {
+  padding-inline-start: calc(var(--lumo-size-m) / 3 + var(--lumo-border-radius-m) / 2);
+  padding-inline-end: calc(var(--lumo-size-m) / 3 + var(--lumo-border-radius-m) / 2);
+  padding-top: 0.5em;
```

Comment:
If I'm not mistaken, hardcoded value  `0.5em` is used because Vaadin does the same. To make it clear, let's add a comment for future me than will forget about it ;)

Reply:
Yes, you're right, I'll add a comment.